internal/machine/lease.go

```diff
@@ -8,14 +8,21 @@ import (
 	"github.com/superfly/flyctl/flaps"
 )
 
-// AcquireLeases works to acquire a leases for each active machine.
+// AcquireAllLeases works to acquire/attach a lease for each active machine.
 // WARNING: Make sure you defer the lease release process.
-func AcquireLeases(ctx context.Context) ([]*api.Machine, error) {
+func AcquireAllLeases(ctx context.Context) ([]*api.Machine, error) {
 	machines, err := ListActive(ctx)
 	if err != nil {
 		return nil, err
 	}
 
+	return AcquireLeases(ctx, machines)
+
+}
+
+// AcquireLeases works to acquire/attach a lease for each machine specified.
+// WARNING: Make sure you defer the lease release process.
+func AcquireLeases(ctx context.Context, machines []*api.Machine) ([]*api.Machine, error) {
```

Comment:
Isn't there a risk of callers forgetting to call release on these returned machines? I return a release function that callers can defer. It's name would make it had to forget.